Update README.md (#3296)
DataDog/integrations-core: wmi_check/README.md

`tag_by` optionally lets you tag each metric with a property from the WMI class you're using. This is only useful when you will have multiple values for your WMI query. The examples below show how you can tag your process metrics with the process name (giving a tag of "name:app_name").

> `constant_tags` optionally lets you tag each metric with a set of fixed values. *Will become deprecated in future versions. Please move towards using tags instead. 

Reviewer: ```suggestion
`constant_tags` optionally lets you tag each metric with a set of fixed values. **Note**: This is deprecated in future versions. Use `tags` instead. 
```
